src/pages/Lend/LendByCategoryPage.vue

```diff
@@ -52,13 +56,8 @@ export default {
 	},
 	data() {
 		return {
-			isAdmin: true,
-			isVisitor: true,
-			itemsInBasket: [],
-			loading: false,
-			rows: [],
+			isAdmin: false,
```

Comment:
won't we need some of these deleted data items for responding to lend-25 events?

Reply:
The lend-25 action is entirely contained within the lend button now